src/core/app.lua

```diff
@@ -236,6 +236,15 @@ function main (options)
       assert(not done, "You can not have both 'duration' and 'done'")
       done = lib.timer(options.duration * 1e9)
    end
+
+   local breathe = breathe
```

Comment:
No :)  It makes a new local variable named `breathe` which is initially bound to the value of the global `breathe` variable.  If we measure latency, then we overwrite our local copy but not the global variable.